Website/Packages/datax-pipeline/src/modules/flowDefinition/flowHelpers.js

```diff
@@ -4,13 +4,6 @@
 // *********************************************************************
 import * as Models from './flowModels';
 
-export function isValidNumberAboveZero(value) {
-    const number = Number(value);
-    const isNumber = !isNaN(number);
-    const isValid = isNumber && number > 0 && value[0] !== '0';
-    return isValid;
-}
-
 export function isValidNumberAboveOrEqualZero(value) {
```

Comment:
If we are moving isValidNumberAboveZero to commons, does it also make sense to move isValidNumberAboveOrEqualZero to commons too? They seem similar/related and other future package developers might benefit from this common helper function.

Reply:
Currently only isValidNumberAboveZero is being used by 2 packages. IMO, we can move the functions to common as needed if they are being used by multiple packages. This is not the only one function in the FlowHelpers.js which could potentially be moved. Thanks!